tests/plugins/templates/manual/insertTemplate.html

```diff
@@ -0,0 +1,10 @@
+<textarea id="editor">
+	<p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr</p>
+</textarea>
+
+<script>
+	CKEDITOR.replace( 'editor', {
+		templates_files: ['/tests//plugins/templates/_assets/test.js'],
```

Comment:
The same here:

```suggestion
		templates_files: ['../_assets/test.js'],
```

or

```suggestion
		templates_files: ['/tests/plugins/templates/_assets/test.js'],
```